corehq/apps/reminders/tests/test_recipient.py

```diff
@@ -65,9 +65,9 @@ def test_recipient_case_owner_location_parent(self):
                 self.assertEqual(reminder.recipient, [parent_location])
 
             # Remove parent location
-            parent_location.delete()
             child_location.parent = None
             child_location.save()
+            parent_location.delete()
```

Comment:
Just updated.  I also put in a `self.addCleanup(parent_location.delete)`, as by the time cleanup happens, child_location will no longer be a child location of parent, so it won't be picked up in the cascading delete.